src/contracts/Test/IbexaKernelTestCase.php

```diff
@@ -31,9 +33,23 @@
  */
 abstract class IbexaKernelTestCase extends KernelTestCase
 {
+    private const FIXTURE_USER_ADMIN_ID = 14;
+    private const FIXTURE_USER_ANONYMOUS_ID = 10;
+
     protected static function getKernelClass(): string
     {
-        return IbexaTestKernel::class;
+        try {
+            return parent::getKernelClass();
+        } catch (LogicException $e) {
+            @trigger_error(sprintf(
+                'You should set the KERNEL_CLASS environment variable to the fully-qualified class name of your '
+                . 'Kernel in phpunit.xml / phpunit.xml.dist or override the "%1$s::createKernel()" or '
+                . '"%1$s::getKernelClass()" method.',
+                static::class
```

Comment:
Why not simply `__METHOD__`?

Reply:
If you look on the message you'll see that it's not always the `__METHOD__` that is suggested. To be completely frank I've copied a similar message in the parent `KernelTestCase`.